README.md

```diff
@@ -31,8 +31,97 @@ Decrypted secrets from `project/config/secrets.json` (or `project/config/secrets
 
 will be accessible via `Rails.application.secrets.some_secret` or `Rails.application.secrets[:some_secret]` on boot. JSON files are loaded once and contents are `deep_merge`'d into your app's existing rails secrets.
 
+Secrets will also be accessible via `Rails.application.credentials`, e.g. `Rails.application.credentials.some_secret` or `Rails.application.credentials[:some_secret]`. To avoid subtle compatibility issues, if a credential already exists, an error will occur.
+
 NOTE: This gem does not decrypt ejson for you. You will need to configure this as part of your deployment pipeline.
 
+## Migrating to credentials
```

Comment:
I suppose because of the human nature or ERB, and ambiguity like that, we're not going to be able to make a migration script then